### ISSUE TITLE
Setup Linux builds for various Ubuntu versions in a PPA

### ISSUE THREAD

**Issue by timabbott** [zulip/zulip-desktop#1]
There's already a debian packaging in debian/  in the repo, so this should be a relatively easy project.

**Comment by timabbott** [comment]
I put these in: https://launchpad.net/~tabbott/+archive/ubuntu/zulip-desktop

I'd love to hear any reports of these working; if so I can post them properly.

**Comment by hakanersu** [comment]
It doesnt work for me.After i enter my server address im getting white blank page.

**Comment by timabbott** [comment]
Does the server have a self-signed certificate?  In that case the problem is #6

**Comment by hakanersu** [comment]
I guess you are right. I'll fix my certificate issue and try again.

**Comment by hakanersu** [comment]
After fixing certificate linux app works fine now. 

Also windows app works with  ``` --site https://yourzulip.com ``` param but my coworkers don't like just first notification hangs on screen.

**Comment by phillip-white-sociomantic** [comment]
@hakanersu How did you fix the certificate issue?

**Comment by hakanersu** [comment]
@phillip-white-sociomantic my certificate wasn't installed correctly so after installed correctly app works fine now.

**Comment by phillip-white-sociomantic** [comment]
@hakanersu That's strange. I believe i have my cert setup correctly. I use a self signed cert installed with:
openssl genrsa -des3 -passout pass:x -out server.pass.key 1024
openssl rsa -passin pass:x -in server.pass.key -out zulip.key
rm server.pass.key
openssl req -new -key zulip.key -out server.csr
openssl x509 -req -days 365 -in server.csr -signkey zulip.key -out zulip.combined-chain.crt
rm server.csr
cp zulip.key /etc/ssl/private/zulip.key; cp zulip.combined-chain.crt /etc/ssl/certs/zulip.combined-chain.crt

Is your cert self signed? How did you generate it?

**Comment by hakanersu** [comment]
@phillip-white-sociomantic Sorry not self signed.

**Comment by timabbott** [comment]
@phillip-white-sociomantic the app doesn't work with self-signed certs right now; see ticket #6

**Comment by lfaraone** [comment]
I'll see if I can't get automatic daily builds up and running on Launchpad this weekend.

**Comment by wolfen351** [comment]
@timabbott please will you update the binary in the https://launchpad.net/~tabbott/+archive/ubuntu/zulip-desktop above?

I'm still getting the white screen, even with a proper cert in place. I have read around these parts there is a --allow-insecure and a --site option? It would be really nice to document the command line switches somewhere?

Also does the above build already contain the 2 switches mentioned above?

**Comment by timabbott** [comment]
Yeah @wolfen351  I actually just uploaded 0.5.1 to my zulip-desktop PPA yesterday.
